ene_008_oil_chemical_spill_incidents/Dockerfile

```diff
@@ -22,7 +21,7 @@ COPY contents/ .
 COPY time.cron .
 
 # Set user for container security
-RUN adduser -D $NAME
+RUN useradd $NAME
```

Comment:
Why use `useradd` here instead of `adduser`?

Reply:
I copied that from another dataset when updating the docker file

Reply:
Probably doesn't matter but we are mostly using `useradd` so this helps us standardize.

Reply:
Updated in PR https://github.com/resource-watch/nrt-scripts/pull/1183